src/index.html

```diff
@@ -50,7 +50,8 @@ <h2 id="meals-count"></h2>
           </ul>
           <h3 class="creator">Created by
             <a href="https://github.com/GraceAbike">GraceAbike</a> and
-            <a href="#"></a> under the MIT licence
+            <a href="https://github.com/keebot-png
+            ">Keenan</a> under the MIT licence
```

Comment:
- [ ] One of your names isn't showing on the page and it might be because of the quotation mark on the 2nd line, kindly fix this issue 👍 

![image](https://user-images.githubusercontent.com/50721479/200130913-ad426a48-377a-463b-ae12-41315d2fee58.png)